mentat/resources/templates/benchmark.jinja

```diff
@@ -0,0 +1,83 @@
+{% from 'transcript.jinja' import transcript_container %}
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Benchmark Results</title>
+        <style>
+            {% include 'css/benchmark.css' %}
+            {% include 'css/transcript.css' %}
+        </style>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/styles/default.min.css">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/highlight.min.js"></script>
+        <script>hljs.initHighlightingOnLoad();</script>
```

Comment:
Oh yeah, that does sort of seem bad in the json code blocks where true mostly means error detected. I'll check. If it's hard I'll just turn off syntax highlighting for json since it doesn't add much anyway